service/src/main/java/com/ge/predix/acs/config/RedisConfig.java

```diff
@@ -43,11 +45,33 @@
     private String redisHost;
     @Value("${REDIS_PORT:6379}")
     private int redisPort;
+    @Value("${REDIS_MIN_ACTIVE:0}")
+    private int minActive;
+    @Value("${REDIS_MAX_ACTIVE:100}")
+    private int maxActive;
+    @Value("${REDIS_MAX_WAIT_TIME:30000}")
```

Comment:
Did you consider using a smaller timeout ? 3 secs ?  Most redis operations are reported as < 1 ms in newrelic. Even 1 second may be enough.